src/pages/userPage/EditUserPage/EditUserPage.tsx

```diff
@@ -181,7 +181,7 @@ export default function () {
       { pattern: allVariantsPattern, message: wrongAllVariantsMessage },
     ],
     placeOfStudy: [
-      { max: 50, message: maxLength(50) },
+      { max: 100, message: maxLength(100) },
```

Comment:
Have you changed maxLength of place of study on the backend?

Reply:
Yes I changed maxLength on backend too and will make pull request today ;)
```suggestion
      { max: 100, message: maxLength(100) },
```

Reply:
ok